src/main/java/uk/gov/companieshouse/items/orders/api/model/DatabaseSequence.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.companieshouse.items.orders.api.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "database_sequences")
+public class DatabaseSequence {
```

Comment:
Can you add a JavaDoc to this please (what's a database sequence for?)?

Reply:
Will do.